modules/ibmz-configure-nbde-with-static-ip.adoc

```diff
@@ -0,0 +1,145 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_z/installing-ibm-z.adoc
+// * installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc
+// * installing/installing_ibm_z/installing-ibm-z-kvm.adoc
+// * installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc
+
+ifeval::["{context}" == "installing-ibm-z"]
+:ibm-z:
+endif::[]
+ifeval::["{context}" == "installing-ibm-z-kvm"]
+:ibm-z-kvm:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-ibm-z"]
+:ibm-z:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-ibm-z-kvm"]
+:ibm-z-kvm:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="configuring-nbde-static-ip-ibmz-linuxone-environment_{context}"]
+= Configuring NBDE with static IP in an {ibmzProductName} or {linuxoneProductName} environment
+
+Enabling NBDE disk encryption in an {ibmzProductName} or {linuxoneProductName} environment requires additional steps, which are described in detail in this section.
+
+.Prerequisites
+
+* You set up the External Tang Server. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption_security-hardening#network-bound-disk-encryption_configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption[Network-bound disk encryption] for instructions.
+* You have installed the `butane` utility.
+* You have reviewed the instructions for how to create machine configs with Butane.
+
+.Procedure
+
+. Create Butane config files for the control plane and compute nodes.
++
+The following example Butane configuration for a control plane node creates a file named `master-storage.bu` for disk encryption:
++
+[source,yaml]
+----
+variant: openshift
+version: 4.13.0
+metadata:
+  name: master-storage
+  labels:
+    machineconfiguration.openshift.io/role: master
+storage:
+  filesystems:
+    - device: /dev/mapper/root
+      format: xfs
+      label: root
+      wipe_filesystem: true
+  luks:
+    - clevis:
+        tang:
+          - thumbprint: QcPr_NHFJammnRCA3fFMVdNBwjs
+            url: http://12.23.21.58:7500
+ifndef::ibm-z-kvm[]
+      device: /dev/disk/by-partlabel/root <1>
+endif::ibm-z-kvm[]
+ifdef::ibm-z-kvm[]
+      device: /dev/disk/by-partlabel/root
+endif::ibm-z-kvm[]
+      label: luks-root
+      name: root
+      wipe_volume: true
+openshift:
+  fips: true
```

Comment:
Are we requiring FIPS mode?  If not, we should footnote that this is optional.

Reply:
Since this has been reinstated, would it make sense to footnote it?

Reply:
Yes it makes sense. I added the callout text we use in the sample install-config.yaml for FIPS